spec/factories/disk.rb

```diff
@@ -1,4 +1,11 @@
 FactoryGirl.define do
   factory :disk do
+    trait(:hdd) do
+      device_name "test_disk"
```

Comment:
I don't see uniqueness validations or find_by's using device_name but should we use sequence here?